src/Tool.java

```diff
@@ -0,0 +1,33 @@
+
+import data.*;
+import engine.*;
+import errors.*;
+import utils.SortType;
+import java.io.File;
+
+public class Tool {
+    private final Config config;
+
+    public Tool(Config cfg) {
+        this.config = cfg;
+    }
+
+    public void launch() throws FileCommandException, NoDataException, NoSortException {
```

Comment:
FileCommandException, NoDataException, NoSortException - there are unchecked exceptions. Throws is not needed
The same for other places with `throws` with these exceptions